tools/gomod-required-updater/internal/updater/config.go

```diff
@@ -0,0 +1,113 @@
+package updater
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+)
+
+type TOMLConfig struct {
+	Modules []string `toml:"modules"`
+}
+
+type Config struct {
```

Comment:
Do we ever expect people to run this locally vs CI? I wonder how much value a config file really provides here vs the complexity it introduces

Reply:
If ran locally, I would expect the hash used to be the previous commit, not the head of develop. Because we always mean to set the commit hash that is the same as what the local replace was resolving to in order to have no effect. We are only trying to aid dependents that import these.

Reply:
No I'm wondering if we really need any complicated git logic at all, or if we should just be looking for parents, and worrying about which to choose in the case of merge commits.

Reply:
Removed the TOML config file option.

Will follow up re: the parents comment.

Reply:
I think I was over-complicating it